static/app/components/textOverflow.tsx

```diff
@@ -4,13 +4,38 @@ type Props = {
   children: React.ReactNode;
   className?: string;
   ['data-test-id']?: string;
+  /**
+   * Change which side of the text is elided.
+   * Default: 'right'
+   *
+   * BROWSER COMPAT:
+   * When set to `left` the intention is for something like: `...xample.com/foo/`
+   * In Chrome & Firefox this is what happens.
+   *
+   * In Safari (July 2022) you will see this instead: `...https://example.co`.
```

Comment:
I'm confused. Is this comment saying this is still an issue, or that this was a problem and is now fixed?